### ISSUE TITLE
implementation using Pytorch geometric

### ISSUE THREAD

**Issue by pvnieo** [graphdeeplearning/benchmarking-gnns#61]
Hi,

Thanks for the great repo and paper.

I'm interested in the CLUSTER and PATTERN experiments, and I want to reproduce them using pytorch-geometric, but I'm having hard time doing this even if I'm using the same loss and accuracy functions, and I tried to use the same hyperparameters.

I want to know if there is any pytorch geometric implementation for these experiments? if so, can you please point me to them? if not, Can you give me general hints on how to reproduce them?

Thank you in advance!

**Comment by chaitjo** [comment]
Hi @pvnieo, thank you for your interest in our work. 

This codebase is not maintained by us, but it is a clone of the DGL benchmarking-gnns repo in PyG. I believe this can be a good place to get started:
https://github.com/karl-zhao/benchmarking-gnns-pyg

Hope it helps!

**Comment by chaitjo** [comment]
Closing for now, feel free to re-open for discussion.

**Comment by pvnieo** [comment]
Hi @chaitjo 

Thanks for the recommendation, I just tested it and worked for me!
